backend/pkg/console/list_messages_integration_test.go

```diff
@@ -361,7 +361,7 @@ func (s *ConsoleIntegrationTestSuite) TestListMessages() {
 				assert.True(ok, "request is not a list offset request: %+T", req)
 
 				require.Len(fetchReq.Topics, 1)
-				assert.Equal(testTopicName, fetchReq.Topics[0].Topic)
+				require.NotEmpty(fetchReq.Topics[0].TopicID)
```

Comment:
Since in this branch we remove MaxVersions in franz-go kgo client creation, fetch req v13+ only uses topic IDs, not names, so the server will see an empty name. The change in the test reflects this.